docs/trusted-web-activity-browser-support.md

```diff
@@ -0,0 +1,42 @@
+Trusted Web Activity Browser Support
+====================================
+
+**Important:** This is an unnofficial list of Trusted Web Activity support across
+browsers. The list is not exhaustive and is maintained by the community.
+Please file an [issue][14] if the information is incorrect, incomplete of if you'd
```

Comment:
File an issue or a pull request? Are we (most likely you) going to end up manually verifying everything?

Reply:
An issue sounds like a good first step? PRs are definitely welcome, but issues may be easier to file and have a quick chat. (I definitely don't plan to manually verify everything)

Reply:
Would it be worth having an Issue template for this specifically?

Reply:
Might be - I'll waiting until we start getting issues on this and then we can map what would be useful to have on the template.